lib/presentation/pages/intro_page.dart

```diff
@@ -22,10 +23,16 @@ class _IntroPageState extends State<IntroPage> {
   IntroState state = IntroState.welcome;
   final Duration selectionTransitionDelay = const Duration(milliseconds: 200);
 
-  void nextPage() => _introKey.currentState?.next();
+  void nextPage() {
+    VibrationController.instance.vibrate(VibrationType.light);
+    _introKey.currentState?.next();
+  }
 
-  void onDone(BuildContext context) => Navigator.of(context)
-      .push(MaterialPageRoute(builder: (context) => HomePage()));
+  void onDone(BuildContext context) {
+    VibrationController.instance.vibrate(VibrationType.light);
+    Navigator.of(context)
+        .push(MaterialPageRoute(builder: (context) => HomePage()));
+  }
```

Comment:
Not needed as you already called vibration on `ready_for_session_page.dart` which call this method